pims/display.py

```diff
@@ -392,22 +393,31 @@ def plot_to_frame(fig, dpi, **imsave_kwargs):
     """
     if mpl is None:
         raise ImportError("This feature requires matplotlib.")
-    try:
-        from PIL import Image
-    except ImportError:
-        raise ImportError("This feature requires PIL/Pillow.")
     from pims import Frame
-    buffer = six.BytesIO()
     if isinstance(fig, mpl.axes.Axes):
         fig = fig.figure
-    fig.savefig(buffer, format='tif', dpi=dpi, **imsave_kwargs)
-    buffer.seek(0)
-    im = np.asarray(Image.open(buffer))
-    buffer.close()
-    return Frame(im)
+
+    agg = fig.canvas.switch_backends(mpl.backends.backend_agg.FigureCanvasAgg)
+    original_bbox = fig.bbox_inches
+    if bbox_inches is not None:
+        fig.bbox_inches = bbox_inches
```

Comment:
This doesn't seem to have an effect. @tacaswell do you no a way to use a tight bounding box in this method? `savefig(..., bbox_inches='tight')` gave nicer results than this implementation.